client/src/components/Dashboard/Dashboard.jsx

```diff
@@ -13,17 +14,49 @@ import { Grid } from "@mui/material";
 import CreateSessionPopup from "../CreateSession/CreateSessionPopup.jsx";
 import SessionMoreInfoPopup from "../SessionMoreInfo/SessionMoreInfoPopup.jsx";
 import { useSelector } from "react-redux";
+import { getSessionsAsync } from "../../redux/session/sessionThunks.js";
+import {useDispatch} from 'react-redux';
 
 export default function Dashboard() {
 
   const [isCreateSessionModalOpen, setIsCreateSessionModalOpen] =
     useState(false);
-  const [isSessionMoreInfoModalOpen, setIsSessionMoreInfoModalOpen] =
-    useState(false);
-  const [selectedSessionId, setSelectedItemId] = useState(null);
-  const profile = useSelector((store) => store.profileReducer).profile;
   const sessions = useSelector((store) => store.sessionReducer).sessions;
 
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getSessionsAsync('near_you'))
+      .then((sessions) => {
+        const near_you = sessions.payload
+        setNearYouSessions(near_you);
+      })
+      .catch((error) => {
+        console.error('Error fetching near you sessions:', error);
+      });
+
+    dispatch(getSessionsAsync('Frisbee'))
+      .then((sessions) => {
+        const frisbeeSessionsData = sessions.payload;
+        setFrisbeeSessions(frisbeeSessionsData);
+      })
+      .catch((error) => {
+        console.error('Error fetching Frisbee sessions:', error);
+      });
+      dispatch(getSessionsAsync('Soccer'))
+      .then((sessions) => {
+        const soccer = sessions.payload;
+        setSoccerSessions(soccer);
+      })
```

Comment:
Maybe we can make these case insensitive? 

There is no guarantee that the user types in the sport name with the first letter capitalized

Reply:
I can surely make them case insensitive - but we are not using it for search here?
Here it is just to show them two different sports.